web/html/src/components/table/TableFilter.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+
+import { Select } from "components/input";
+import { Form } from "components/input/Form";
+import { SelectSearchField } from "components/table/SelectSearchField";
+
+const renderSearchField = (props) => {
+  const { field } = props;
```

Comment:
This works, but feels a bit weird. I think it would make sense to either destruct everything (`const { field, criteria, placeholder, onSearch, name } = props`), or destruct nothing and use `props.field` in the line below this one. I think the latter makes more sense, but either way is fine with me as long as it's consistent.